docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -307,6 +317,21 @@ The Observability AI Assistant connector is called when the alert fires and when
 
 To learn more about alerting, actions, and connectors, refer to <<create-alerts>>.
 
+[discrete]
+[[obs-ai-settings]]
+== AI Assistant Settings
+
+Access the AI Assistant Settings page:
+
+* From *{stack-manage-app}* -> *Kibana* -> *AI Assistants* -> *Elastic AI Assistant for Observability*.
+* From the *More ations* menu inside the AI Assistant window.
+
+The AI Assistant Settings page contains the following tabs:
+
+* *Settings*: configure the main AI Assistant settings, which are explained directly within the interface.
+* *Knowledge base*: <<obs-ai-kb-ui,manage knowledge base entries>>. 
+* *Search Connectors*: provides a link to {kib} *Search* -> *Content* -> *Connectors* UI for connectors configuration.
+
```

Comment:
```suggestion
* *Settings*: Configures the main AI Assistant settings, which are explained directly within the interface.
* *Knowledge base*: Manages <<obs-ai-kb-ui,knowledge base entries>>. 
* *Search Connectors*: Provides a link to {kib} *Search* -> *Content* -> *Connectors* UI for connectors configuration.
```
Capitalize the first word after the colon.
Keep the tense consistent, either infinite or 3rd person singular.